GenderPayGap.WebUI/Controllers/Account/ChangePasswordController.cs

```diff
@@ -61,6 +62,13 @@ public IActionResult ChangePasswordPost(ChangePasswordViewModel viewModel)
             
             // Check that passwords are valid
             ValidatePasswords(viewModel, currentUser);
+            if (currentUser.ResetAttempts == Global.MaxLoginAttempts)
+            {
+                currentUser.ResetAttempts = 0;
+                dataRepository.SaveChanges();
+                StatusMessageHelper.SetStatusMessage(Response, "You have been logged out for security reasons", Url.Action("LoggedOut", "Login"));
+                return LoginHelper.Logout(HttpContext, RedirectToAction("LoggedOut", "Login"));

```

Comment:
can the user attempt to carry on changing their password after they are logged out as we've set their reset attempts back to 0?

Reply:
yes but their account is locked for 30 minutes - same behavior when introducing the wrong credentials at login

Reply:
Ah brill, wasn't sure where the lock out logic was